lib/slas.js

```diff
@@ -79,10 +79,10 @@ const slas = {
                     handleCLIError('Could not get tenant: ', e.message, asJson)
                 }
             },
-            list: async (shortcode, asJson) => {
+            list: async (shortcode, tenantId, asJson) => {
                 let result
                 try {
-                    result = await slas.api.tenant.list(shortcode)
+                    result = await slas.api.tenant.list(shortcode, tenantId)
```

Comment:
I think the real issue here is that the `get tenants` endpoint doesn't exist anymore but this at least returns a result for 1 single tenant... but i can revert out so we can get the other fixes in